backend/src/controllers/inventory-item-controller.ts

```diff
@@ -45,7 +52,7 @@ export const createInventoryItem = async (req: Request, res: Response) => {
     }
 };
 
-export const getInventoryItems = async (req: Request, res: Response) => {
+export const getInventoryItems = async (req: CustomRequest, res: Response) => {
```

Comment:
Let the user add a `?itemId=____` filter to filter Inventory Items by `itemId`

Reply:
We can talk about this in the meeting, but what do you mean by having itemId? Do I add it to the custom request?